lib/sdn_epc/ofsh_calls.ex

```diff
@@ -4,29 +4,31 @@ defmodule SdnEpc.OfshCalls do
 
   # :ofs_handler callbacks
 
-  def init(_mode, _ip, datapatch_id, _features, _version, _connection, _opts) do
+  def init(_mode, _ip, datapath_id, _features, _version, _connection, _opts) do
     {:ok, ofpc_sup} = start_ofp_channel_sup()
-    open_ofp_channel ofpc_sup
-    SdnEpc.Forwarder.subscribe_messages_from_switch datapatch_id, :packet_in
-    {:ok, datapatch_id}
+    open_ofp_channel(ofpc_sup)
+    SdnEpc.Forwarder.save_datapath_id(datapath_id)
+    SdnEpc.Forwarder.subscribe_messages_from_switch(datapath_id, :packet_in)
```

Comment:
Make the `SdnEpc.Forwarder.subscribe_messages_from_switch/2` accept list of messages types.